Sources/OHHTTPStubsSwift/OHHTTPStubsSwift.swift

```diff
@@ -174,7 +174,82 @@ public extension HTTPStubsResponse {
   }
 #endif
 
+/**
```

Comment:
I wonder if it is good to duplicate the same documentation so many times. Maybe it is better to say it is equivalent to calling the `stub` function with a block that returns <whatever the function do>.

Reply:
It follows the style of the code in this file.